packages/eas-cli/src/update/configure.ts

```diff
@@ -252,6 +254,71 @@ async function ensureEASUpdatesIsConfiguredNativelyAsync(
   }
 }
 
+/**
+ * Make sure EAS Build profiles are configured to work with EAS Update by adding channels to build profiles.
+ */
+
+async function ensureEASUpdateIsConfiguredInEasConfigAsync({
```

Comment:
```suggestion
async function ensureEASUpdateIsConfiguredInEasJsonAsync({
```

Reply:
And rename all variables in this function from `easConfig` to `easJson`. We shouldn't use more than one name for the same thing. It's really confusing for new people reading the code.